tests/PHPUnit/Unit/Metrics/FormatterTest.php

```diff
@@ -190,9 +190,9 @@ public function getPrettyTimeFromSecondsData()
             array(100, array('1 min 40s', '00:01:40')),
             array(3600, array('1 hours 0 min', '01:00:00')),
             array(3700, array('1 hours 1 min', '01:01:40')),
-            array(86400 + 3600 * 10, array('1 days 10 hours', '34:00:00')),
-            array(86400 * 365, array('365 days 0 hours', '8760:00:00')),
-            array((86400 * (365.25 + 10)), array('1 years 10 days', '9006:00:00')),
+            array(86400 + 3600 * 10, array('1 days 10 hours', '1 days 10:00:00')),
```

Comment:
@mattab @tsteur I wanted to know more about <code>Intl_OneDay</code>.
Can I know where is this implemented? 
Will it just add the string day or will it compute intelligently whether to use day or days. 
If it just adds a simple string days, then I'd like to help and do something which intelligently checks if it should be days or day.